src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig

```diff
@@ -32,8 +32,8 @@
                     <div class="row">
                       <div class="col-sm-3">
                         <div id="mediaTree">
-                          <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-                            <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
+                          <button class="btn btn-default btn-sm btn-link tree-collapsed" data-role="toggle-js-tree-collapse">
```

Comment:
Do you have a screenshot of this? If i try this out the button background becomes white, that does not seem like an enchainment to me.